src/Components/Filters/Filters.css

```diff
@@ -0,0 +1,32 @@
+.Filters {
```

Comment:
Request!
Please, use lower-case for class names, for ex.: `.filters {}`
PS: it's because we use such code convention in our current project, which you are going to join :)